scripts/install/opt/seagate/cortx/motr/sanity/motr_sanity.sh

```diff
@@ -182,172 +190,302 @@ generate_endpoints()
 	fi
 
 	unused_port_get "$base_port"
-	local_endpoint="${IP}:12345:44:$port"
-	echo "Local endpoint: $local_endpoint"
+	if [ "$XPRT" = "lnet" ]; then
+		local_endpoint="${IP}:12345:44:$port"
+		ha_endpoint="${IP}:12345:45:1"
+	else
+		local_endpoint="${IP}@$port"
+		ha_endpoint="${IP}@2001"
+	fi
 
-	ha_endpoint="${IP}:12345:45:1"
+	echo "Local endpoint: $local_endpoint"
 	echo "HA endpoint: $ha_endpoint"
 
-	profile_fid='<0x7000000000000001:0>'
+	profile_fid='0x7000000000000001:0'
 	echo "Profile FID: $profile_fid"
 
-	ios1_fid='<0x7200000000000001:3>'
+	ios1_fid='0x7200000000000001:3'
 	echo "Ioservice 1 FID: $ios1_fid"
 
-	process_fid='<0x7200000000000000:0>'
+	process_fid='0x7200000000000000:0'
 	echo "Process FID: $process_fid"
 }
 
 error_handling()
 {
 	echo "$1 with rc = $2"
 	stop_singlenode
-	exit $2
+	exit "$2"
 }
 
 object_io_test()
 {
-	echo "Running Object IO tests"
+	local rc
+
+	echo "***** Running Object IO tests *****"
+
 	obj_id1="20:20"
 	obj_id2="20:22"
 	blk_size="4k"
 	blk_count="200"
 	src_file="$SANITY_SANDBOX_DIR/src"
 	dest_file="$SANITY_SANDBOX_DIR/dest"
-        echo $blk_size $blk_count
-	dd if="/dev/urandom" of=$src_file bs=$blk_size count=$blk_count || {
+
+	echo "----- Size: $blk_size, Count: $blk_count -----"
+
+	dd if="/dev/urandom" of="$src_file" bs="$blk_size" count="$blk_count" || {
 		error_handling "dd command failed" $?
 	}
+
 	endpoint_opts="-l $local_endpoint -H $ha_endpoint -p $profile_fid \
 		       -P $process_fid"
-	rm -f $dest_file
-	m0cp $endpoint_opts -o $obj_id1 -s $blk_size -c $blk_count $src_file || {
-		error_handling "Failed to write object" $?
-	}
-	m0cat $endpoint_opts -o $obj_id1 -s $blk_size -c $blk_count $dest_file || {
-		error_handling "Failed to read object" $?
-	}
-	diff $src_file $dest_file || {
+
+	rm -f "$dest_file"
+
+	echo "Writing to object $obj_id1....."
+	eval "m0cp $endpoint_opts -o $obj_id1 -s $blk_size -c $blk_count $src_file"
+	rc=$?
+	if [ $rc -ne 0 ]; then
```

Comment:
Why is the style changed? Was there any issue with the old code ?

Reply:
No issue with old code. But now it is shifted more towards right. So decided to split it. I can use old style if that's preferred.

Reply:
Used earlier style now.